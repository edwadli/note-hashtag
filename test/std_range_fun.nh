x = Range 1 3
for i in x do PrintEndline (StringOfInt i)

if Range 1 1 == int{} then PrintEndline "empty range" else PrintEndline "you sucks"
