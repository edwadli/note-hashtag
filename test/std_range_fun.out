1
2
empty range
